Kinvey/Kinvey/Client.swift

```diff
@@ -72,7 +72,15 @@ open class Client: NSObject, NSCoding, Credential {
         }
     }
     
-    internal var urlSession = URLSession(configuration: URLSessionConfiguration.default) {
+    internal static let urlSessionConfiguration: URLSessionConfiguration = {
```

Comment:
I just reverted this, I think I changed to test something and forgot to revert. Thanks for notice this.